### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,5 +1,71 @@
-# This is a file to standardize editor settings: http://EditorConfig.org
```
Comment (sarina) [inline .editorconfig:1]: This was previously not generated by `edx_lint`. Should this stay or be reverted?

**Comment by sarina** [comment]
Rebecca/Tim if one of you could look at it I'd really appreciate it :)

File: .editorconfig (hunk 1 above)
Comment (timmc-edx) [reply .editorconfig:1]: Probably want whatever edx-lint is writing out.

File: README.rst [hunk 2]
```diff
@@ -339,30 +339,30 @@ Some common service combinations include:
...
+.. _frontend-app-learning: https://github.com/openedx/frontend-app-learning
+.. _frontend-app-library-authoring: https://github.com/openedx/frontend-app-library-authoring
+.. _frontend-app-course-authoring: https://github.com/openedx/frontend-app-course-authoring
+.. _frontend-app-account: https://github.com/openedx/frontend-app-account
 .. _frontend-app-profile: https://github.com/openedx/frontend-app-profile
 .. _frontend-app-authn: https://github.com/openedx/frontend-app-authn
-.. _xqueue: https://github.com/edx/xqueue
-.. _coursegraph: https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/coursegraph
+.. _xqueue: https://github.com/openedx/xqueue
+.. _coursegraph: https://github.com/openedx/edx-platform/tree/master/openedx/core/djangoapps/coursegraph
```
Comment (timmc-edx) [inline README.rst]: [inform] Dead link here, but not really in the scope of this PR to fix.

File: repo.sh [hunk 3]
```diff
@@ -21,66 +21,66 @@ fi
...
-    "https://github.com/edx/frontend-app-program-console.git"
-    "https://github.com/edx/frontend-app-account.git"
+    "https://github.com/openedx/frontend-app-course-authoring.git"
+    "https://github.com/openedx/frontend-app-learning.git"
+    "https://github.com/openedx/frontend-app-library-authoring.git"
+    "https://github.com/openedx/registrar.git"
+    "https://github.com/openedx/frontend-app-program-console.git"
+    "https://github.com/openedx/frontend-app-account.git"
     "https://github.com/openedx/frontend-app-profile.git"
     "https://github.com/edx/frontend-app-ora-grading.git"
```
Comment (timmc-edx) [inline repo.sh:49]: Is this repo something that still needs to move over, or is that staying in the edx org for now?

File: repo.sh [hunk 4]
```diff
@@ -21,66 +21,66 @@ fi
...
 ssh_repos=(
-    "git@github.com:edx/course-discovery.git"
-    "git@github.com:edx/credentials.git"
-    "git@github.com:edx/cs_comments_service.git"
-    "git@github.com:edx/ecommerce.git"
+    "git@github.com:openedx/course-discovery.git"
+    "git@github.com:openedx/credentials.git"
+    "git@github.com:openedx/cs_comments_service.git"
+    "git@github.com:openedx/ecommerce.git"
     "git@github.com:edx/edx-e2e-tests.git"
```
Comment (timmc-edx) [inline repo.sh:57]: Not sure if this one will be moving.

File: repo.sh [hunk 5]
```diff
@@ -21,66 +21,66 @@ fi
...
+    "git@github.com:openedx/frontend-app-program-console.git"
+    "git@github.com:openedx/frontend-app-account.git"
     "git@github.com:openedx/frontend-app-profile.git"
     "git@github.com:edx/frontend-app-ora-grading.git"
 )
 
 private_repos=(
     # Needed to run whitelabel tests.
-    "https://github.com/edx/edx-themes.git"
+    "https://github.com/openedx/edx-themes.git"
```
Comment (timmc-edx) [inline repo.sh]: This one hasn't actually moved over.

File: repo.sh (hunk 4 above)
Comment (sarina) [reply repo.sh:57]: I didn't change this one - it's still in the `edx` org

File: repo.sh (hunk 3 above)
Comment (sarina) [reply repo.sh:49]: I only made edits that reflect the current state of affairs. I think it's up to 2U to decide if they want to move it.

Comment (timmc-edx) [reply repo.sh:49]: Yeah, I'm just keeping an eye out for repos that we might have missed, since there have been a couple stragglers. :-)

**Comment by sarina** [comment]
Fixed the `edx-themes` path and updated the Coursegraph path.

**Review by timmc-edx — APPROVED** [review]

**Review by timmc-edx — APPROVED** [review]

**Pull request merged**